CMakePresets.json

```diff
@@ -18,6 +18,17 @@
         "dev": true,
         "uninitialized": true
       }
+    },
+    {
+      "name": "coverage",
+      "displayName": "Build for coverage analysis",
+      "generator": "Unix Makefiles",
+      "cacheVariables": {
+        "CMAKE_C_COMPILER": "gcc",
```

Comment:
https://clang.llvm.org/docs/SourceBasedCodeCoverage.html clang supports this too, so no need to set a default here, I think.

Reply:
Doesn't clang require the use of the llvm-cov tool, which is currently undocumented in the README.md?